drafter.gyp

```diff
@@ -39,5 +43,61 @@
         "ext/snowcrash/snowcrash.gyp:libsos",
       ],
     },
+
+    {
+      'target_name': 'test-libdrafter',
+      'type': 'executable',
+      'include_dirs': [
+        'src',
+        'test',
+        'test/vendor/Catch/include',
+        "ext/snowcrash/src",
+        "ext/snowcrash/ext/markdown-parser/src",
+        "ext/snowcrash/ext/markdown-parser/ext/sundown/src",
+        "ext/snowcrash/ext/sos/src",
+      ],
+      'sources': [
+        "test/test-main.cc",
+        "test/test-SerializeSourceAnnotations.cc",
+        "test/test-cdrafter.cc",
+      ],
+      'dependencies': [
+        "libdrafter",
+        "ext/snowcrash/snowcrash.gyp:libsnowcrash",
+        "ext/snowcrash/snowcrash.gyp:libmarkdownparser",
+        "ext/snowcrash/snowcrash.gyp:libsundown",
+        "ext/snowcrash/snowcrash.gyp:libsos",
+      ]
+    },
+
+    {
+      "target_name": "drafter",
+      "type": "executable",
+      "sources": [
+        "src/main.cc",
+        "src/config.cc",
+        "src/config.h",
+        "src/reporting.cc",
+        "src/reporting.h",
+      ],
+
+      # FIXME: replace by direct dependecies
```

Comment:
I will resolve it later.

There are some troubles while include `snowcrash/common.gypi` and dependencies are no working as declared there:
https://code.google.com/p/gyp/wiki/InputFormatReference#Dependencies_and_Dependents

Will fix after cleanup snowcrash